extensions/elytron-security/runtime/src/main/java/io/quarkus/elytron/security/runtime/UndertowTokenCredential.java

```diff
@@ -0,0 +1,8 @@
+package io.quarkus.elytron.security.runtime;
+
+import io.undertow.security.idm.Credential;
+
+public interface UndertowTokenCredential extends Credential {
```

Comment:
Same question here, will it work with the Undertow dependency ?

Reply:
This will be followed up in a later PR. The idea is to move all the HTTP related stuff to vert.x so it can be shared

Reply:
Hi, I fixed a typo in my question, I meant **without** the Undertow dependency :-). But I think your answer clarifies it anyway